src/index.jsx

```diff
@@ -49,8 +50,23 @@ window.addEventListener("message", listenForEvalFramePortReady, false);
 handleServerVariables(store);
 handleInitialJsmd(store);
 handleReportViewModeInitialization(store);
-checkForAutosave(store);
-checkForServerAutosave(store);
+
+// re-apply any locally saved changes if applicable
+checkForLocalAutoSave(store);
```

Comment:
Reading this bit of code, I wonder if something like `applyLocallySavedChanges(store)` might be a better name for what this is doing. That may pigeonhole this fcn for later use, though - I leave it to you.

Reply:
Yeah, this sounds right. I'll rename the function before landing.